shipyard_blueprints/notion/pyproject.toml

```diff
@@ -10,7 +10,7 @@ readme = "README.md"
 python = "^3.9"
 requests = "2.31"
 pandas = "1.5.3"
-shipyard-templates = "^0.4.1a0"
+shipyard-templates = "^0.4.1a1"
```

Comment:
is there a pre release or was this intended to be `0.4.1`

Reply:
This is intended to be 0.4.1. I decided to publish so I could finalize Notion [](edited-with-Axolo)